prdoc/pr_1378.prdoc

```diff
@@ -0,0 +1,27 @@
+title: Construct Runtime V2 - An outer macro approach to define the runtime
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Introduces `#[frame_support::runtime]` that can be attached to a mod to define a runtime. The items 
+      in this mod can be attached to the following attributes to define the key components of the runtime.
+      1. `#[runtime::runtime]` attached to a struct defines the main runtime
+      2. `#[runtime::derive]` attached to the runtime struct defines the types generated by the runtime
+      3. `#[runtime::pallet_index]` must be attached to a pallet to define its index
+      4. `#[runtime::disable_call]` can be optionally attached to a pallet to disable its calls
+      5. `#[runtime::disable_unsigned]` can be optionally attached to a pallet to disable unsigned calls
```

Comment:
What does this mean? Currently the behaviour is that a pallet always rejects unsigned calls that do not pass `validate_unsigned`.  
Does this mean that it will not even generate the enum variants for this or what?

Reply:
The pallet will generate it but it won't be included in the aggregates generated by the runtime macro.